agora_analytica/flask/templates/main.html

```diff
@@ -23,29 +23,82 @@
 
         <div class="navbar-collapse" id="main-navigation">
             <ul class="navbar-nav">
+
                 <li class="nav-item dropdown">
-                    <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown">Vaalipiirit</a>
+                    <a href="#" select id = "dropdownVaalip" class="nav-link dropdown-toggle" data-toggle="dropdown" >Vaalipiirit</a>
                     <ul class="dropdown-menu">
-                        <li><a href="#" class="small" data-value="option1" tabIndex="-1"><input
-                                    type="checkbox" />&nbsp;Vaalipiiri A</a></li>
-                        <li><a href="#" class="small" data-value="option2" tabIndex="-1"><input
-                                    type="checkbox" />&nbsp;Vaalipiiri B</a></li>
-                        <li><a href="#" class="small" data-value="option3" tabIndex="-1"><input
-                                    type="checkbox" />&nbsp;Vaalipiiri C</a></li>
+                        <li><a href="#" class="small" data-value="option1" tabIndex="-1">&nbsp;<input
+                                    type="checkbox" />&nbsp;Helsinki</a></li>
```

Comment:
Selkokieliset tekstit olisi hyvä ajaa juuri sen flask-babelin läpi, jolloin sovelluksen pystyy muuntamaan kansainväliseksi.
```suggestion
                                    type="checkbox" />&nbsp;{{ _("Helsinki")}}</a></li>
```